### PR TITLE
Modify Obtaining API Keys page to the correct syntax.

### PR DESCRIPTION
**Pull request by lychiang**
## Description

Modify Obtaining API Keys page to the correct syntax

## Motivation and Context

API Syntax changed

## How Has This Been Tested?

Tested it in the local 

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ x] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [x ] I have added tests to cover my changes if appropriate.
- x[ ] All new and existing tests passed.

### CODE REVIEW

**Comment by lychiang** [comment]
@csestito Please review and approve/merge the change. thank you!

File: products/expedition/docs/expedition_workflow_searchrule.md [hunk 1]
```diff
@@ -0,0 +1,766 @@
+---
+id: expedition_workflow_searchrule
+title: List security rules contain specific subnet
```
Comment (csestito) [inline products/expedition/docs/expedition_workflow_searchrule.md:3]: Should this be "List security rules containing a specific subnet" ?

**Review by csestito — APPROVED** [review]
Just one suggested edit

File: products/expedition/docs/expedition_workflow_searchrule.md (hunk 1 above)
Comment (lychiang) [reply products/expedition/docs/expedition_workflow_searchrule.md:3]: Sure , please go ahead to change the wording

**Review by csestito — APPROVED** [review]

**Pull request merged**